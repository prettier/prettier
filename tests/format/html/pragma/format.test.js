runFormatTest(import.meta, ["html"], { requirePragma: true });
runFormatTest(import.meta, ["html"], { insertPragma: true });
