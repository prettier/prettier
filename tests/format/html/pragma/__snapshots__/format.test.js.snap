// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-pragma.html - {"insertPragma":true} format 1`] = `
====================================options=====================================
insertPragma: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================


<!-- @not-a-pragma -->
<!doctype html>
<html>
            </html>

=====================================output=====================================
<!-- @format -->

<!-- @not-a-pragma -->
<!doctype html>
<html></html>

================================================================================
`;

exports[`no-pragma.html - {"requirePragma":true} format 1`] = `
====================================options=====================================
parsers: ["html"]
printWidth: 80
requirePragma: true
                                                                                | printWidth
=====================================input======================================


<!-- @not-a-pragma -->
<!doctype html>
<html>
            </html>

=====================================output=====================================


<!-- @not-a-pragma -->
<!doctype html>
<html>
            </html>

================================================================================
`;

exports[`with-pragma.html - {"insertPragma":true} format 1`] = `
====================================options=====================================
insertPragma: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================


<!-- @format -->
<!doctype html>
<html>
            </html>

=====================================output=====================================
<!-- @format -->
<!doctype html>
<html></html>

================================================================================
`;

exports[`with-pragma.html - {"requirePragma":true} format 1`] = `
====================================options=====================================
parsers: ["html"]
printWidth: 80
requirePragma: true
                                                                                | printWidth
=====================================input======================================


<!-- @format -->
<!doctype html>
<html>
            </html>

=====================================output=====================================
<!-- @format -->
<!doctype html>
<html></html>

================================================================================
`;

exports[`with-pragma-2.html - {"insertPragma":true} format 1`] = `
====================================options=====================================
insertPragma: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================


<!-- @prettier -->
<!doctype html>
<html>
            </html>

=====================================output=====================================
<!-- @prettier -->
<!doctype html>
<html></html>

================================================================================
`;

exports[`with-pragma-2.html - {"requirePragma":true} format 1`] = `
====================================options=====================================
parsers: ["html"]
printWidth: 80
requirePragma: true
                                                                                | printWidth
=====================================input======================================


<!-- @prettier -->
<!doctype html>
<html>
            </html>

=====================================output=====================================
<!-- @prettier -->
<!doctype html>
<html></html>

================================================================================
`;
